crane-cloud/frontend
# add apps search funtionality

# Description

Integrate app search functionality with the backend

## Type of change



## Trello Ticket ID

https://trello.com/c/i5kvX9zq
https://trello.com/c/4O9MNJFs

## How Can This Been Tested?

Check the branch out and try to search through applications
Note: run yarn before  testing as this PR involves use of a simple library the adds a caching data structure
First time you load the apps or each page or new search word, it may take longer to load but once a similar page, or word
is used on the endpoint again, it will use cached data making the search of already added words faster and navigating through 
pages

## Checklist:


# Screenshots


https://github.com/crane-cloud/frontend/assets/69305400/a5f1dea8-9c02-4954-9253-4c1174ce9457
